flow.js

```diff
@@ -84,6 +88,9 @@ class Flow {
         let condition
         if(typeof value === 'object'){
           condition = nextFlowStep.find(o => o.condition === value.value)
+          if(condition === undefined) {
+            condition = nextFlowStep.find(o => o.condition === value.type)
```

Comment:
This is the same as [line 90](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/73/files#diff-4f43f537bc3e655255f7b417ad7e89ad1c24015fa0b2139cc7411ec1a3bb9419R90) and I don't think that it is changing the value of `condition` - if line 90 assigns null/`undefined`, then so will this line.

Please explain why this line is needed or delete it.